dwh/query_lib.py

```diff
@@ -62,9 +63,9 @@ def patient_query_factory(runner: Runner, data_source: str) -> PatientQuery:
     ValueError: When the input `data_source` is malformed or not implemented.
   """
   if runner == Runner.SPARK:
-    return _SparkPatientQuery(data_source)
+    return _SparkPatientQuery(data_source, code_system)
```

Comment:
does this provide ability of passing more than one coding system?

Reply:
No, right now we have assumed that one coding system is used everywhere.